test/correctness/simd_op_check.cpp

```diff
@@ -1455,23 +1460,23 @@ int main(int argc, char **argv) {
         in_i64 = ImageParam(Int(64), 1, "in_i64"),
         in_u64 = ImageParam(UInt(64), 1, "in_u64")
     };
-
-    for (ImageParam p : image_params) {
+    if (can_run_code()) {
```

Comment:
Exactly. it is just that I thought changing to not use realize would be a more significant change than this.